src/main/java/Item.java

```diff
@@ -21,13 +22,14 @@ public class Item {
 
 
 
-    public Item(String name, double price, double desiredPrice, String url, String itemDescription, String[] tags, int reviewCount, double reviewStars){
+    public Item(String name, double price, double desiredPrice, String url, String imageUrl, String itemDescription, String[] tags, int reviewCount, double reviewStars){
         this.itemName = name;
```

Comment:
Instead of putting the new field in the middle of the constructor, can you add it to the end, as it might mess up the database item parsing